Gruntfile.js

```diff
@@ -50,6 +80,44 @@ module.exports = function (grunt) {
         });
     };
 
+    // Base distribution configuration
+    // This should specify only the options for include/exclude;
+    // a parallel "minify" set with options.compress: true
+    // will be generated for the final configuration
+    var baseDistributions =
```

Comment:
I prefer to have the configuration as part of the gruntInitConfig as this is where you'd expect to be modifying configuration if you wanted to add or change the distributions. I think the distributions task should be responsible for creating the minified distributions as well as the source ones.